dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,183 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+
+== Overview
+
+For this example, we are going to create a Jenkins Pipeline that will build, deploy, and verify a Node.js/MongoDB
+application using the https://github.com/openshift/nodejs-ex/blob/master/openshift/templates/nodejs-mongodb.json[nodejs-mongodb.json] template.
+
+[[setting-up-jenkins]]
+
+== Setting up Jenkins
+
+Creating the Jenkins master via the command line:
+
+----
+  // select the project that you want to use
+  $ oc project <project_name>
+  // if you want to use persistent storage, use jenkins-persistent instead
+  $ oc new-app jenkins-ephemeral
+----
+
+[NOTE]
+====
+If Jenkins auto-provisioning is enabled on your cluster, and you don't need to make any customizations to the
+Jenkins master, you can skip the above step.
+====
+
+[[create-pipeline-build-config]]
+
+== Create the Pipeline Build Config
+
+Now that the Jenkins master is up and running, we will create a build config that employs the Jenkins pipeline strategy
+to build, deploy, and scale our Node.js/MongoDB example application.
+
+Create a file named *sample-nodejs-pipeline.yaml* with the below content:
+
+[source,yaml]
+----
+kind: "BuildConfig"
+apiVersion: "v1"
+metadata:
+  name: "sample-nodejs-pipeline"
+spec:
+  strategy:
+    jenkinsPipelineStrategy:
+      jenkinsfile: ...
```

Comment:
the above instructions imply this BuildConfig should be created verbatim, but it's incomplete...

Reply:
something like "\<pipeline content from below\>" would be better.

Reply:
Updated in next push